payroll-processor-functions/src/payroll-processor-functions/Features/Payrolls/PayrollEntity.cs

```diff
@@ -0,0 +1,81 @@
+using System;
+using Microsoft.WindowsAzure.Storage.Table;
+
+namespace PayrollProcessor.Functions.Features.Payrolls
+{
+    public class PayrollEntity : TableEntity
+    {
+        public DateTimeOffset CheckDate { get; set; }
+        public Guid EmployeeId { get; set; }
+        public double GrossPayroll { get; set; }
+        public string PayrollPeriod { get; set; } = "";
```

Comment:
Setting the `= ""` for `string` props to make the nullable reference type checking happy.